packages/date-time-picker/theme/lumo/vaadin-date-time-picker-styles.js

```diff
@@ -14,42 +14,25 @@ registerStyles(
   css`
     :host {
       margin-right: 2px;
+      --vaadin-input-container-top-end-radius: 0;
+      --vaadin-input-container-bottom-end-radius: 0;
```

Comment:
With the current implementation in #5267 does not actually map these props to the logical properties, so the RTL styles would not work as expected. The `border-radius` shorthand property is not direction aware, and always maps the properties to the same corners (top-left, top-right, bottom-right, bottom-left).

Reply:
Good point, thanks. I will update to not use shorthand.